source/tests/test_model_compression.py

```diff
@@ -17,6 +18,16 @@ def _file_delete(file) :
     if os.path.exists(file):
         os.remove(file)
 
+def _subprocess_run(command):
+    popen = sp.Popen(command, shell=True, stdout=sp.PIPE, stderr=sp.STDOUT)
```

Comment:
I prefer setting `shell` to `False` and using `command.split()` instead of `command`. Using a shell [is not safe](https://docs.python.org/3/library/subprocess.html#security-considerations).

Reply:
done